[2023-05-04T20:45:11.986+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-04T20:40:00+00:00 [queued]>
[2023-05-04T20:45:11.991+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-04T20:40:00+00:00 [queued]>
[2023-05-04T20:45:11.991+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T20:45:11.991+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-04T20:45:11.991+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T20:45:12.001+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): delete_file> on 2023-05-04 20:40:00+00:00
[2023-05-04T20:45:12.004+0000] {standard_task_runner.py:55} INFO - Started process 334481 to run task
[2023-05-04T20:45:12.006+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'chatgpt_dag', 'delete_file', 'scheduled__2023-05-04T20:40:00+00:00', '--job-id', '4007', '--raw', '--subdir', 'DAGS_FOLDER/chatgpt_dag.py', '--cfg-path', '/tmp/tmp00p6cgl_']
[2023-05-04T20:45:12.006+0000] {standard_task_runner.py:83} INFO - Job 4007: Subtask delete_file
[2023-05-04T20:45:12.036+0000] {task_command.py:389} INFO - Running <TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-04T20:40:00+00:00 [running]> on host trivia-machine.us-central1-a.c.triviapractice-381517.internal
[2023-05-04T20:45:12.081+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=richc
AIRFLOW_CTX_DAG_ID=chatgpt_dag
AIRFLOW_CTX_TASK_ID=delete_file
AIRFLOW_CTX_EXECUTION_DATE=2023-05-04T20:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-04T20:40:00+00:00
[2023-05-04T20:45:12.085+0000] {logging_mixin.py:137} INFO - /home/rich2/airflow/scripts/chatgpt_output/uscompanies_2023-05-04_20-45-08.json deleted successfully
[2023-05-04T20:45:12.086+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-04T20:45:12.088+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=chatgpt_dag, task_id=delete_file, execution_date=20230504T204000, start_date=20230504T204511, end_date=20230504T204512
[2023-05-04T20:45:12.137+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-04T20:45:12.144+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
